From 3f302cb16ebfefade17b3ea90c5ed258c9150f3a Mon Sep 17 00:00:00 2001
From: Mike Gilbert <floppym@gentoo.org>
Date: Fri, 1 Mar 2019 21:37:52 -0500
Subject: [PATCH] gcc: pass -mavx2 instead of -march=core-avx2

Resolves a build failure when QMAKE_CXXFLAGS contains -march=native
and the compiler is running on a CPU which does not support AVX2.

Bug: https://bugs.gentoo.org/672946
---
 mkspecs/common/gcc-base.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkspecs/common/gcc-base.conf b/mkspecs/common/gcc-base.conf
index c2669e4833..44aa96af38 100644
--- a/mkspecs/common/gcc-base.conf
+++ b/mkspecs/common/gcc-base.conf
@@ -109,7 +109,7 @@ QMAKE_CFLAGS_MIPS_DSP  += -mdsp
 QMAKE_CFLAGS_MIPS_DSPR2 += -mdspr2
 
 # -march=haswell is supported as of GCC 4.9 and Clang 3.6
-QMAKE_CFLAGS_ARCH_HASWELL           = -march=core-avx2
+QMAKE_CFLAGS_ARCH_HASWELL           = -mavx2
 
 # Wrapper tools that understand .o/.a files with GIMPLE instead of machine code
 QMAKE_AR_LTCG           = gcc-ar cqs
-- 
2.21.0.rc1

